src/scripts/eos-analysis

```diff
@@ -386,6 +392,20 @@ def cmd_find_mode(args):
 
         info('Starting minimization from point: {}'.format(starting_point))
 
+        _bfp = analysis.optimize(starting_point)
+        _gof = eos.GoodnessOfFit(analysis.log_posterior)
+        _chi2 = _gof.total_chi_square()
+        if _chi2 < min_chi2:
+            gof = _gof
+            bfp = _bfp
+            min_chi2 = _chi2
+    elif args.seed:
+        seed = args.seed
+        rng = RNG(seed)
+        starting_point = [p.inverse_cdf(rng.uniform(0, 1)) for p in analysis.log_posterior.log_priors()]
```

Comment:
Isn't this duplicating code? I would have just called ``analysis.optimize("random", rng=rng)``

Reply:
You are correct, I don't know what I had in mind...